### ISSUE TITLE
A11y_Azure Tools for VS Code_Azure Function_Create New Project_Screen Reader: Voiceover is not announcing the placeholder text and role for an input field.

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/vscode-azurefunctions#3820]
### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #AILimited; #Azure Tools for VS Code; #BM-AzureToolforVSCode-MAC-AUG2023; #DesktopApp;  #A11yWCAG1.3.1; #Info and Relationships; #Element:Input; #Screen reader; #Voiceover; #AINotInScope; #Device; #MAC;

### OS Details:
Mac OS Ventura 13.5

### Environment Details:
Application Name: Azure Tools for VS Code

### Repro Steps:
1. Turn on Voiceover.
2.Open Visual studio code
3.Click on extension button and search for "Azure Functions"
4. Click on install and move to the "Azure" button
5. Click on "Create New Project Button" from the toolbar.
6.Verify Screen reader is reading the placeholder text "Select a language", "select Subscription" "Select the folder containing your function project" etc. and role of an input field while creating "New project", "Functions" and on "Subscription".

### Actual Result:
Voice over is not announcing the placeholder text "Select a language", "select Subscription" "Select the folder containing your function project" etc. and role of an input field while creating "New project", "Functions" and on "Subscription".

**Note:** Similar issue is observed in "Create function" and " Select subscription" scenarios also.

### Expected Result:
Voiceover should announce the the placeholder text "Select a language", "select Subscription" "Select the folder containing your function project" etc. and role of an input field while creating "New project", "Functions" and on "Subscription".

### Attachment:
![A11y_Plaeholder-and-control-role-_1_](https://github.com/microsoft/vscode-azurefunctions/assets/98530069/af4e4fd6-aebc-4a33-befd-271772386c04)

**Comment by MicroFish91** [comment]
Tracking: https://github.com/microsoft/vscode/issues/195723

**Comment by squ33z3r** [comment]
Add Tag #A11yCAIOH

**Comment by kupatkar99** [comment]
#Closed;
Verified the issue in environment:
Visual studio Code Version: 1.82.1 (Universal)
Issue is fixed. hence closing the bug please find added attcahment for reference.
<img width="1440" alt="MicrosoftTeams-image" src="https://github.com/microsoft/vscode-azurefunctions/assets/98530069/51205881-b48b-4fe7-998b-5cdce88a78c0">
